backend/otel/otel.go

```diff
@@ -82,32 +82,6 @@ func projectToInt(projectID string) (int, error) {
 	return 0, e.New(fmt.Sprintf("invalid project id %s", projectID))
 }
 
-func getAttributesMap(resourceAttributes, eventAttributes map[string]any) map[string]string {
-	attributesMap := make(map[string]string)
-	for _, m := range []map[string]any{resourceAttributes, eventAttributes} {
-		for k, v := range m {
-			for _, attr := range highlight.InternalAttributes {
-				if k == attr {
-					continue
```

Comment:
This was the bug...we were doing a `continue` for the inner loop (`for _, attr := range highlight.InternalAttributes`) but we needed to `continue` out of the outer loop `for k, v := range m {`